src/primitives/add_operation.cpp

```diff
@@ -175,7 +175,7 @@ namespace phylanx { namespace primitives
     // implement '+' for all possible combinations of lhs and rhs
     hpx::future<ir::node_data<double>> add_operation::eval()
     {
-        return hpx::dataflow(hpx::util::unwrapping(
+        return hpx::dataflow(hpx::util::unwrapped(
```

Comment:
This has been recently renamed. What version of HPX should we target with Phylanx? Would it be ok to stick with HPX master for the foreseeable future?

Reply:
my poor git utilization strikes again! apologies, will amend.

Reply:
@hkaiser do i need to do anything to drop that last commit for 'hpx::util::unwrapping'?

Reply:
@ct-clmsn it should be fine to remove the commit from your local branch (reset to b18e6332b070e17a7f32297ea0a5f88e91d765f8) and force-push to github.